handlers/product-switch-api/test/contributionToSupporterPlusIntegration.test.ts

```diff
@@ -74,6 +74,76 @@ describe('product-switching behaviour', () => {
 
 		expect(result).toMatchObject(expectedResult);
 	});
+	it('can preview an annual recurring contribution switch with 50% discount', async () => {
```

Comment:
worth adding a gap to make it easier to pick out the different tests
```suggestion

	it('can preview an annual recurring contribution switch with 50% discount', async () => {
```

Reply:
probably not important